README.md

```diff
@@ -681,6 +682,25 @@ $server = new Server(new MiddlewareRunner([
 ]));
 ```
 
+#### CompressionGzipMiddleware
+
+One of the built-in middleware is the `RequestBodyBufferMiddleware` which
+can be used to automatically compress the response when it is allowed
+by the incoming [request](#request) (`Accept-Encoding: gzip`). This will result in
+a [response](#response) with an `Content-Encoding: gzip` header in combination with
+the new calculated `Content-Length` header.
+
+Usage:
+
+```php
+$middlewares = new MiddlewareRunner([
+    new \React\Http\Middleware\CompressionGzipMiddleware(),
```

Comment:
No need to use the FQCN here, just `CompressionGzipMiddleware` (like you do with `MiddlewareRunner` 😃 ).